src/wakefs/fuse.cpp

```diff
@@ -217,16 +233,56 @@ bool run_in_fuse(fuse_args &args, int &status, std::string &result_json) {
   (void)close(STDOUT_FILENO);
   (void)close(STDERR_FILENO);
 
-  do waitpid(pid, &status, 0);
-  while (WIFSTOPPED(status));
+  bool has_timeout = (bool)args.command_timeout;
+  pid_t timeout_pid = -1;
+
+  // Launch timer process
+  if (has_timeout) {
+    timeout_pid = fork();
+    if (timeout_pid < 0) {
+      std::cerr << "wakebox: failed to fork timeout process" << std::endl;
+      exit(1);
+    }
+
+    if (timeout_pid == 0) {
+      sleep(*args.command_timeout);
```

Comment:
```suggestion
      prctl(PR_SET_NAME, "wb-timer", 0, 0, 0);
      sleep(*args.command_timeout);
```

Reply:
Example of inspecting using the thread names

Running this spec:
```
{
  "command": [
    "/usr/bin/sleep", "5"
  ],
  "command_timeout": 50,
  "isolate-pids": true
}
```

In one terminal:
```
$ echo $$
605615
$ unshare -mrp --mount-proc -f ./bin/wakebox -p spec.json
```
Then while that runs, in a second:
```
$ pstree -p 605615
bash(605615)───unshare(1903513)───wakebox(1903514)─┬─fuse-waked(1903517)
                                                   ├─wb-mount-ns(1903521)───wb-pid-ns(1903523)───sleep(1903524)
                                                   └─wb-timer(1903522)
```